.formatter.exs

```diff
@@ -3,7 +3,7 @@
   inputs: [
     "{mix,.formatter}.exs",
     "{config,test}/**/*.{ex,exs}",
-    "lib/{belfrage, belfrage_web}/**/*.{ex,exs}"
```

Comment:
the reason we did this was to avoid the formatter to change the Routefile, wouldn't be the case now?

Reply:
> the reason we did this was to avoid the formatter to change the Routefile, wouldn't be the case now?

This is OriginSimulator calling 😁 i.e. no route file here.